projects/storefrontstyles/scss/components/myaccount/_cx-my-account-v2-notification-preference.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../theme';
+
+%cx-my-account-v2-notification-preference {
+  .np-content-center {
+    -ms-flex-pack: center !important;
+    justify-content: center !important;
+  }
+
+  .np-flex {
+    display: -ms-flexbox !important;
+    display: flex !important;
+  }
+
+  .np-row {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    right: -($font-size-base * 0.9375);
+    left: -($font-size-base * 0.9375);
+  }
+
+  .header {
+    width: $font-size-base * 14.375;
+    height: $font-size-base * 1.875;
+    font-weight: var(--cx-font-weight-semi);
+    font-size: $h3-font-size;
```

Comment:
* Please use our `—cx-color` variables for colours and `@include type()` mixin for predefined fonts. We are no longer using Bootstrap variables.
* Please make sure all your CSS code follows the guidelines provided on this wiki page https://wiki.one.int.sap/wiki/display/spar/UI+Styling+and+Accessibility+Onboarding 
* Please reach out to Spartacus Theme Owner if any doubts and to follow up with code review: @morganm58